api/v1beta1/foundationdbcluster_types.go

```diff
@@ -328,6 +335,11 @@ type FoundationDBClusterStatus struct {
 	// sidecar conf in the config map even when the latest version should not
 	// require it.
 	NeedsSidecarConfInConfigMap bool `json:"needsSidecarConfInConfigMap,omitempty"`
+
+	// StorageServersPerDisk defines the current count of storage servers per disk
+	// this value should be StorageServersPerPod * ProcessCount for storage class.
+	// If the value differs the reconcile phase is not finished
+	StorageServersPerDisk map[string]bool `json:"storageServersPerDisk,omitempty"`
```

Comment:
What are the keys in this map? I'm guessing instance IDs? I generally prefer to make that explicit in the documentation.

Reply:
From context later on, it seems like the keys in this map are a string representation of the number of storage servers?

Reply:
Total drive by comment here, but map types are not advised in Kubernetes APIs and are against convention 😄 https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps